cmd/backup/backup.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"sort"
+	"time"
+
+	azstorage "github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift/openshift-azure/pkg/log"
+	"github.com/openshift/openshift-azure/pkg/util/managedcluster"
+)
+
+const (
+	cMaxBackups = 6
+	copyRetries = 3
+	interval    = 5 * time.Second
+)
+
+var (
+	logLevel    = flag.String("loglevel", "Debug", "Valid values are Debug, Info, Warning, Error")
+	blobName    = flag.String("blobname", "", "Name of the blob")
```

Comment:
can these be whittled down to `./backup save` and `./backup restore blobname >file` ? (don't care about the choice of words, just avoiding the options)